packages/flac/flac.0.1.1/opam

```diff
@@ -4,18 +4,15 @@ build: [
   ["./configure" "--prefix" lib "--mandir" man]
   [make "all"]
 ]
-depends: ["ocaml" "ocamlfind" "ogg" "base-unsafe-string"]
-install: [make "install"]
-depexts: [
-  ["flac-dev"] {os-distribution = "alpine"}
-  ["flac"] {os-distribution = "arch"}
-  ["flac-devel"] {os-distribution = "centos"}
-  ["flac-devel"] {os-distribution = "fedora"}
-  ["flac-devel"] {os-family = "suse"}
-  ["libflac-dev"] {os-family = "debian"}
-  ["flac"] {os-distribution = "nixos"}
-  ["flac"] {os = "macos" & os-distribution = "homebrew"}
+depends: [
+  "ocaml"
+  "ocamlfind"
+  "ogg"
+  "base-unsafe-string"
+  "conf-pkg-config" {build}
```

Comment:
Should `conf-pkg-config` be removed since it's already required by `conf-libflac`?
```suggestion
```

Reply:
It's always better to be explicit about things the package use explicitly, `conf-libflac` might use a different method of detection in the future, we never know.

Unless the old packages did not use `pkg-config` I would suggest to keep `conf-pkg-config`. Is that the case?

Reply:
That makes total sense. The packages does indeed use `pkg-config` so it makes sense to keep it there.